packages/eslint-plugin-cli/rules/app-commands-requirements.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path')
+
+/**
+ * Every App Command should have a default export that extends `AppCommand`
+ *
+ * This way we control that all commands are constrained by what's defined in the `AppCommand` abstract class.
+ * Such as the return type for the run() function.
+ */
+module.exports = {
```

Comment:
Could you use a JSDoc comment to add some type checking to this? Like: https://github.com/Shopify/cli/blob/aa8b871813d93197b6e2925a8eb5d4aa87f96267/packages/eslint-plugin-cli/rules/no-inline-graphql.js#L75

Makes it easier to later modify the rule.